docs/content/Select.mdx

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
`Select` wasn't added to `nav.yml` and is therefore not present in the sidebar menu. Was this intentional @mperrotti? You can only find this doc page via the search atm.

Reply:
Not intentional. Adding now...